shm/Game.cpp

```diff
@@ -98,7 +109,7 @@ Coordinates Game::getTravelLocation() {
     return Coordinates(X, Y);
 }
 
-void Game::advanceTimeTraveling(size_t distance) {
+void Game::advanceTimeTraveling(int distance) {
```

Comment:
I don't really like the idea that "distance" can be a negative value.

Reply:
True distance in and of itself cannot be negative. But if it's an uneven value then subtracting an even value from it can switch it to negative.  This solution seems cleaner to me than a couple of ifs